documentation/src/main/asciidoc/user_guide/hibernate-ogm.adoc

```diff
@@ -0,0 +1,47 @@
+==  Hibernate OGM
+Hibernate can perform CRUD operations directly on an Infinispan cluster.
+Hibernate OGM is an extension of the popular Hibernate ORM project which makes
+the Hibernate API suited to interact with NoSQL databases such as Infinispan.
+
+From the perspective of the Hibernate user, it might be interesting to replace
+the RDBMS with Infinispan to benefit from the better scalability and higher
+performance on CRUD operations; the drawback is that Infinispan - not being a
+relational database - can not run complex relational queries.
+
+From the perspective of the Infinispan user, using Hibernate OGM can simplify
+the learning curve as the JPA API is quite simple to use and already well known
+by many developers.
+
+But Hibernate OGM brings many more benefits, not least experience with using
+Infinispan and transparently addressing several problems you might not even
```

Comment:
What does this mean "not least experience with using Infinispan"? Needs rephrasing and clarification,